services/storage/tests/utils.py

```diff
@@ -10,7 +18,14 @@
 SECRET_KEY = '12345678'
 
 BUCKET_NAME ="simcore-testing"
+USER_ID = '0'
 
+def has_datcore_tokens()->bool:
+    token = os.environ.get("BF_API_KEY", "none")
```

Comment:
Same as above, I think there is not need to use the string ``"none"`` instead of the default ``None``